jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
```

Comment:
Interesting.  Users should be able to edit an issue without adding a comment (change watchers, change assigned, etc).  If you leave the comment body blank go-jira should strip that field from the document submitted to the jira api.  There might be a bug with that, the document is fixed in the yamlFixup routine in util.go.